src/sentry/snuba/metrics/query_builder.py

```diff
@@ -282,33 +233,32 @@ def to_metrics_query(self) -> MetricsQuery:
             histogram_to=self.histogram_to,
         )
 
-    def _parse_orderby(self, query_params):
-        orderby = query_params.getlist("orderBy", [])
-        if not orderby:
+    @staticmethod
+    def _parse_orderby(query_params):
+        orderbys = query_params.getlist("orderBy", [])
+        if not orderbys:
             return None
-        elif len(orderby) > 1:
-            raise InvalidParams("Only one 'orderBy' is supported")
```

Comment:
This is the only piece of validation that I would leave in `QueryDefinition`. That way, `MetricsQuery.orderby` can keep its type.